code/datums/elements/ai_retaliate.dm

```diff
@@ -3,12 +3,18 @@
  * The AI controller is responsible for doing anything with that information.
  */
 /datum/element/ai_retaliate
+	/// What mob type the parent calls
+	var/mob/living/mob_type
+	/// In what range the parent calls the same mobs to retaliate
```

Comment:
This should not be handled by the retaliate component but by the AI itself. (E.g. "If I have enemies in my retaliate list, I should call for help")

Reply:
but maybe @tralezab disagrees with me? :D

Reply:
I did it like this since it is essentially the input. I could also try making a child element for it but I'll wait to tralezab to respond.

Reply:
Fikou told me of a callback idea I really liked, but this would also make for a good datum ai_behavior as well, keeping the retaliate clean of "optional behavior" which can stack up over time. If @CapybaraExtravagante is happy to help you turn this into an ai behavior, we should go for that. Otherwise, an optional callback to the gnome where it can rouse its friends works. But try ai behavior first!

Reply:
I used a callback, reason being the AI behavior did weird for me and it having a cool-down to check for each others gnome lists every two seconds or so didn't seem efficient.

Reply:
Thats because you werent supposed to check the other gnomes list but instead only your own list :p

Reply:
I only used that for the callback I made now, back when I used a behavior for it, the gnomes constantly stood still for a second to check for the targeting behavior I think